### ISSUE TITLE
Tasks are occasionally executed multiple times

### ISSUE THREAD

**Issue by bonniegee** [RockefellerArchiveCenter/zodiac#169]
## Describe the bug
If a task takes a long time (which happens somewhat frequently with services in Pictor and Fornax with digitzed content), celery executes the task queue multiple times. This means that services run over themselves because they are started multiple times at once.

## To reproduce
1. Run services  in the pipeline that take a long time
2. Look at the celery logs

## Expected behavior
Celery executes the task queue one at a time.

## Error message or screenshots
From the celery logs:
```
[2022-02-03 21:48:00,116: INFO/MainProcess] Task gateway.tasks.queue_services[b92c3df1-38e2-4ecd-9fa7-ea340e690b52] received
[2022-02-03 21:48:00,159: INFO/ForkPoolWorker-1] Task gateway.tasks.queue_services[b92c3df1-38e2-4ecd-9fa7-ea340e690b52] succeeded in 0.04119482636451721s: {'detail': {'services': [{...}, {...}, {...}, {...}, {...}]}}
[2022-02-03 21:49:00,005: INFO/MainProcess] Task gateway.tasks.queue_services[3697b827-ac05-4420-bb64-343ef9f6ea98] received
[2022-02-03 21:50:00,100: INFO/MainProcess] Task gateway.tasks.queue_services[70155fb3-fbc8-453a-98bc-c91bcdcd1157] received
[2022-02-03 21:51:00,010: INFO/MainProcess] Task gateway.tasks.queue_services[6360fafa-40a1-4126-9fc1-913f54bf2126] received
[2022-02-03 21:51:06,613: INFO/ForkPoolWorker-2] Task gateway.tasks.queue_services[3697b827-ac05-4420-bb64-343ef9f6ea98] succeeded in 0.04446461796760559s: {'detail': {'services': [{...}, {...}, {...}, {...}, {...}]}}
[2022-02-03 21:51:06,647: INFO/ForkPoolWorker-2] Task gateway.tasks.queue_services[70155fb3-fbc8-453a-98bc-c91bcdcd1157] succeeded in 0.03273032605648041s: {'detail': {'services': [{...}, {...}, {...}, {...}, {...}]}}
[2022-02-03 21:51:06,686: INFO/ForkPoolWorker-2] Task gateway.tasks.queue_services[6360fafa-40a1-4126-9fc1-913f54bf2126] succeeded in 0.033785656094551086s: {'detail': {'services': [{...}, {...}, {...}, {...}, {...}]}}
[2022-02-03 21:52:00,008: INFO/MainProcess] Task gateway.tasks.queue_services[a888f281-6cb6-4d88-b1af-9c7dbd9dbeb8] received
[2022-02-03 21:52:00,053: INFO/ForkPoolWorker-2] Task gateway.tasks.queue_services[a888f281-6cb6-4d88-b1af-9c7dbd9dbeb8] succeeded in 0.043983474373817444s: {'detail': {'services': [{...}, {...}, {...}, {...}, {...}]}}
```

## Impact on your work
This behavior occasionally causes errors in some other microservices, which range from annoying (getting Archivematica error messages in Fornax) to requiring manual intervention (Ursa Major sending incomplete bags to Pictor).

## Additional context
[This issue](https://github.com/celery/celery/issues/3430) and [this section](https://docs.celeryproject.org/en/stable/tutorials/task-cookbook.html#ensuring-a-task-is-only-executed-one-at-a-time) of the docs may be useful.

**Comment by bonniegee** [comment]
After deploying on dev and production and running several (very large) bags through, I can confirm this was fixed by #170 🥳

**Comment by bonniegee** [comment]
So this issue was _not_ fixed by either #170 or #177 and is still persisting. This was noticed by errors thrown in Zodiac, but if you look at the worker log for 2022-05-12 20:32:12-2022-05-12 20:32:15 you can see the duplicated tasks.

**Comment by bonniegee** [comment]
The next approach we're going to try is to rethink task triggering so we're creating individual celery tasks (rather than a task which creates other tasks). We could likely do this by shifting the queue_requests task to a regular cron job.
